pkg/virtctl/credentials/addkey/addkey_test.go

```diff
@@ -59,7 +58,7 @@ var _ = Describe("Credentials add-ssh-key", func() {
 			return false, secret, nil
 		})
 
-		vmi = api.NewMinimalVMI(vmName)
+		vmi = libvmi.New(libvmi.WithNamespace(metav1.NamespaceDefault))
```

Comment:
I think it's a good opportunity to create something for `WithAccessCredential`. I see it will be useful in so many places in future